docs/GroupDefs.hpp

```diff
@@ -196,9 +196,22 @@ The following type aliases are not always required, but will be in some
    integers.
 2. `using explicit_single_actions` is set to a typelist of Actions
    that are not part of the algorithm but can be called remotely at any time,
-   similar in spirit to a member function. Each Action must specify a member
-   type alias called `apply_args` whose elements are the types of the data that
-   is sent to the %Parallel Component and passed into the Action.
+   similar in spirit to a member function.  An Action used as an explicit
+   single action has the following restrictions:
+   1. It must specify a member type alias called `apply_args` whose
+      elements are the types of additional arguments passed into the
+      Action's `apply` function (these additional arguments are passed
+      after the ParallelComponent argument).  If there are no additional
+      arguments, use an empty typelist.
+   2. It's returned DataBox must have the same type as its input DataBox
+      (`db::mutate` can be used to change values, but not types, of
+      things in the DataBox).
```

Comment:
This should mention, at least parenthetically, the exception for the first initialization action that goes from an empty DataBox to `initial_databox`.